### PR TITLE
Adding announcement CBL-Mariner - removing CBL-D

### PR DESCRIPTION
**Pull request by theJasonHelmick**
Replacing CBL-D update announcement with CBL-Mariner update.

### CODE REVIEW

File: CBL-Mariner-Update.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+– Mariner" (aka CBL-Mariner). That may sound unfamiliar – this is not a standalone distribution, but
+a Microsoft project which tracks Red Hat very closely. The primary difference between Red Hat and
+CBL-Mariner is that Microsoft compiles all the packages included in the CBL-Mariner repository
+internally. This helps guard against supply chain attacks.
+
+Tooling has been updated to reflect the new base image CBL-Mariner and should have little impact on
+customers. If your scenario has been affected by these changes, please contact Azure support, or
+create an issue in the [Cloud Shell repository](https://github.com/Azure/CloudShell/issues).
+
+For a full list of the package versions installed via APT, you can run `apt list --installed`.
```
Comment (darrentu) [inline CBL-Mariner-Update.md:18]: tdnf list installed

Comment (dsajanice) [reply CBL-Mariner-Update.md:18]: LGTM

**Comment by maertendMSFT** [comment]
@theJasonHelmick, dont merge this.  We will post this as a blog instead and link to the release which has the included tools listed.

**Comment by theJasonHelmick** [comment]
Will not merge 😉

From: Danny Maertens ***@***.***>
Date: Wednesday, July 27, 2022 at 2:08 PM
To: Azure/CloudShell ***@***.***>
Cc: Jason Helmick ***@***.***>, Mention ***@***.***>
Subject: Re: [Azure/CloudShell] Adding announcement CBL-Mariner - removing CBL-D (PR #224)

@theJasonHelmick<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FtheJasonHelmick&data=05%7C01%7Cjason.helmick%40microsoft.com%7Cf147bcc7b6a840ca33c408da701428ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637945529143814406%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=vN0O99zUt7v2jYMPBtQqM8E7SpVp3cY4CDtrSOHYxsI%3D&reserved=0>, dont merge this. We will post this as a blog instead and link to the release which has the included tools listed.

**Comment by maertendMSFT** [comment]
Closing as content has been released as a blog and release notes

**Pull request closed without merging**